tasks/mocha.js

```diff
@@ -281,7 +283,29 @@ module.exports = function(grunt) {
 
       var stats = helpers.reduceStats(testStats);
 
-      if (stats.failures === 0) {
```

Comment:
I'm about to make a 1.0 release. It seems reasonable to fail if 0 tests are run, regardless of configuration. If you're not willing to make those changes, I can make them.